libc/docs/configure.rst

```diff
@@ -54,6 +54,7 @@ to learn about the defaults for your platform and target.
     - ``LIBC_CONF_SCANF_DISABLE_INDEX_MODE``: Disable index mode in the scanf format string.
 * **"setjmp" options**
     - ``LIBC_CONF_SETJMP_AARCH64_RESTORE_PLATFORM_REGISTER``: Make setjmp save the value of x18, and longjmp restore it. The AArch64 ABI delegates this register to platform ABIs, which can choose whether to make it caller-saved.
+    - ``LIBC_CONF_SETJMP_ENABLE_FORTIFICATION``: Protect jmp_buf by masking its contents and storing a simple checksum, to make it harder for an attacker to read meaningful information from a jmp_buf or to modify it. This is only supported on x86-64 Linux.
```

Comment:
Actually, does the word `ENABLE` need to be in this name? `LIBC_CONF_SETJMP_FORTIFICATION` seems clear enough, and shorter. If you set that to `ON` or `OFF`, nobody's going to be confused about whether you're enabling or disabling the feature.

Reply:
fixed